website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -159,12 +187,177 @@ status:
 
   # Generated instance profile name from "role"
   instanceProfile: "${CLUSTER_NAME}-0123456778901234567789"
+  conditions:
```

Comment:
Should we add a section for the condition fields?

Reply:
I added descriptions of status.conditions and status.resources.